docs/manage/partitions/flex/flex-pricing-faq.md

```diff
@@ -75,4 +75,10 @@ The table below should give you a sense of how the number of partitions you use
 We recommend you configure partitions to have less than 5 TB per day flowing into them. This practice:
 
 * Reduces the cost of each query.
-* Results in faster queries, since less data has to be scanned.  
\ No newline at end of file
+* Results in faster queries, since less data has to be scanned. 
+
+## How are preview queries charged?
+
+While the current impact is minimal, preview queries, such as those displayed during creation or monitoring processes, may incur charges depending on the context. For instance, during monitor creation, a query may run automatically to display data, potentially resulting in charges for the queried period. Although these charges are typically negligible, they can affect cost optimization.
+
+To avoid unnecessary charges, consider implementing user-initiated query execution. In scenarios like the Alert Response Page (ARP) and Emulate search, preview queries may incur repetitive charges. Optimizing these processes can help reduce costs and enhance user experience.
```

Comment:
Didn't understand "consider implementing user-initiated query execution" and "Optimizing these processes". How would users do this?

Reply:
Hi @tarunk2 , great catch! Here's a revised paragraph. Is there anything users can do to avoid these charges?

To mitigate potential unnecessary charges, consider <!-- placeholder -->. In scenarios like the Alert Response Page (ARP) and Emulate search, preview queries may incur repetitive charges. <!-- Optimizing these processes can help reduce costs and enhance user experience. -->

Reply:
Not really. Since charges are going to be negligible, there is nothing users can do to turn them off at this moment.

Reply:
Thank you for the clarification! Reduced this just to one sentence. What do you think? https://github.com/SumoLogic/sumologic-documentation/pull/4133/files

Reply:
Looks good now, thanks!

Reply:
sweet - thank you so much for your help with this